tests/smoke/support/api/http.ts

```diff
@@ -8,18 +8,20 @@ export const request = async ({
   method,
   path,
   body,
-  user
+  user,
+  versionApi
 }: {
   method: 'POST' | 'DELETE' | 'PUT' | 'GET'
   path: string
   body?: BodyInit
   user?: User
+  versionApi: string
```

Comment:
can we make v2.php as a default and versionApi optional (versionApi?)

Reply:
@fschade thanks you for the review. I corrected it